README.md

```diff
@@ -68,6 +68,27 @@ bimtap.ExportStream(resultExport, AnyBitmap.ImageFormat.Jpeg, 100);
 System.Drawing.Bitmap image = new System.Drawing.Bitmap("FILE_PATH");
 IronSoftware.Drawing.AnyBitmap anyBitmap = image;
 anyBitmap.SaveAs("result-from-casting.png");
+
+
+// Create multiple page Tiff image by Array of AnyBitmap
+List<AnyBitmap> bitmaps = new List<AnyBitmap>()
+{
+    AnyBitmap.FromFile("FILE_PATH_1"),
+	AnyBitmap.FromFile("FILE_PATH_2")
+};
+AnyBitmap anyBitmap = AnyBitmap.CreateMultiFrameTiff(bitmaps);
+
+// Create multiple page Tiff image by Array of fully qualified file path
```

Comment:
Would use: Creates an AnyBitmap from an list of Tiff images

Reply:
Like above, perhaps we can include an example of a fully qualified path if we want to label this example with it.